src/tactic/sort.lean

```diff
@@ -0,0 +1,186 @@
+/-
+Copyright (c) 2022 Arthur Paulino, Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Arthur Paulino, Damiano Testa
+-/
+import data.polynomial.degree.definitions
```

Comment:
This import line should be replaced for something more basic like `tactic.⋯`

Reply:
In order to do this, we need to move the heuristics about monomials somewhere else, since otherwise `polynomial.monomial` complains.  I think that it makes sense to do this, but it is also nice to be able to work with a specialized heuristic for monomials.